File: CODE FILES/Mobile/MyApiary/app/src/main/java/projects/myapiary/models/Apiary.kt
package projects.myapiary.models

import com.google.gson.annotations.SerializedName

data class Apiary (
    @SerializedName("id")
    val id: Int,
    @SerializedName("name")
    val Name: String,
    @SerializedName("address")
    val Address: String,
    @SerializedName("connectionWire")
    val ConnectionWire: Boolean,
    @SerializedName("connectionWireless")
    val ConnectionWireless: Boolean,
    @SerializedName("countOfBeehives")
    val CountOfBeehives: Int,
    @SerializedName("notifications")
    val Notifications: Boolean
)

File: CODE FILES/Backend/MyApiary/Resource.Api/Models/Apiary.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Resource.Api.Models
{
    public class Apiary
    {
        [Required]
        public int Id { get; set; }

        [Required]
        public string Name { get; set; }

        [Required]
        public string Address { get; set; }

        [Required]
        public bool ConnectionWire { get; set; }
        [Required]
        public bool ConnectionWireless { get; set; }

        [Required]
        public int CountOfBeehives { get; set; }

        [Required]
        public bool Notifications { get; set; }
    }
}
